assets/files.wxs

```diff
@@ -190,8 +190,8 @@
             <Component Id="cmp2B60F4AC2E8D95A558C3E121A6252D37" Guid="{C361734D-C0E0-47E2-B867-82878D0F0E4C}">
                 <File Id="fil8E8E3BF1C97D92DFEB4FC97D778F238F" KeyPath="yes" Source="$(env.ProductSourcePath)\System.Runtime.CompilerServices.VisualC.dll" />
             </Component>
-            <Component Id="cmp8777CA81FBD5C5FAC86993893043793D" Guid="{B2D9982B-D46A-4D69-AA16-CC7EC49D357F}">
-                <File Id="fil9A6CE1B34698E9E81BC367B9E9CEBF25" KeyPath="yes" Source="$(env.ProductSourcePath)\sos_$(var.FileArchitecture)_$(var.FileArchitecture)_4.6.26515.07.dll" />
+-            <Component Id="cmp94bba02944344967a5b4abedb5e5ad21" Guid="{0e836bc9-94c6-42f2-b213-2990d06172be}">
```

Comment:
You don't have to update the GUIDs/IDs.  Not sure, but the wix error might be expecting GUIDs to be all uppercase?

Reply:
If the file name changed, you do have to change either the Id or the GUID.

Reply:
Should I fix only case or something else?

Reply:
I'm guessing that the case is what you need to fix.  The error is not very useful when compared to the change.